### ISSUE TITLE
does not contain a CMakeLists.txt file.

### ISSUE THREAD

**Issue by hugoanderson5** [bedrock-viz/homebrew-tap#1]
Trying to install bedrock-viz on macOS Catalina 10.15.2 but getting this error. Any ideas?

```
LAZERBOOK:~$ brew tap bedrock-viz/homebrew-tap
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> New Formulae
flint                       git-remote-gcrypt           libffcall                   meilisearch
==> Updated Formulae
aliyun-cli        bitwise           dhall-bash        gmic              okteto            skaffold
angular-cli       bundletool        erlang            groonga           osslsigncode      sqlcipher
arduino-cli       calicoctl         eslint            h3                pulumi            starship
aspectj           cargo-completion  exploitdb         i2p               rsnapshot         weaver
bazelisk          crc32c            file-formula      libmagic          rust              wireguard-tools
bedtools          deno              fio               libvpx            rustup-init       xrootd
bind              devspace          git-extras        minikube          serverless
==> Deleted Formulae
boost-python@1.59

LAZERBOOK:~$ brew install bedrock-viz
==> Installing bedrock-viz from bedrock-viz/tap
==> Downloading https://github.com/bedrock-viz/bedrock-viz/archive/v0.1.1.tar.gz
Already downloaded: /Users/troy/Library/Caches/Homebrew/downloads/65be093c08de1ec871db91999d655710c4fe8bd0200c0251d3f799594c848af4--bedrock-viz-0.1.1.tar.gz
Warning: Cannot verify integrity of 65be093c08de1ec871db91999d655710c4fe8bd0200c0251d3f799594c848af4--bedrock-viz-0.1.1.tar.gz
A checksum was not provided for this resource.
For your reference the SHA-256 is: 8622a5c3ae3cb7034297c399ef32e481a3b7c802d16f5fbb5e3f8531d15a597b
==> cmake .. -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/bedrock-viz/v0.1.1 -DCMAKE_BUILD_TYPE=Release -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_
Last 15 lines from /Users/troy/Library/Logs/Homebrew/bedrock-viz/01.cmake:
    /tmp/bedrock-viz-20191221-29204-xqk483/bedrock-viz-0.1.1/third_party/spdlog

  does not contain a CMakeLists.txt file.


-- Found ZLIB: /usr/lib/libz.dylib (found version "1.2.11") 
-- Found PNG: /usr/local/lib/libpng.dylib (found version "1.6.37") 
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Found LibXml2: /usr/local/opt/libxml2/lib/libxml2.dylib (found version "2.9.10") 
-- Configuring incomplete, errors occurred!
See also "/tmp/bedrock-viz-20191221-29204-xqk483/bedrock-viz-0.1.1/build/CMakeFiles/CMakeOutput.log".

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/bedrock-viz/homebrew-tap/issues
```

**Comment by jasper-wan** [comment]
Fixed in e5ff492. Please try again.

**Comment by hugoanderson5** [comment]
That fixed it, thank you! I did run into another error but will create a new ticket.
